config/kibana.yml

```diff
@@ -61,6 +61,11 @@
 # must be a positive integer.
 # elasticsearch.requestTimeout: 30000
 
+# List of Kibana client-side headers to send to Elasticsearch. To send *no* client-side
+# headers, set this value to null, not []. Setting it to [] will send the default list
```

Comment:
Commenting out the configuration is the standard way to use the default value for configurations, so I recommend ditching the extra logic for `null` altogether.  So `[]` sends no headers, commenting out sends the default (which is just authorization).